package.json

```diff
@@ -37,6 +37,7 @@
     "can-define": "^1.0.0",
     "can-observation": "^3.0.0",
     "can-stream": "^0.0.5",
+    "can-stream-kefir": "git://github.com/canjs/can-stream-kefir.git#mixin",
```

Comment:
My understanding of #17 is that we _don’t_ want this module to depend on `can-stream-kefir` or `kefir`. Is my understanding wrong?